docs/src/development.md

```diff
@@ -471,14 +471,16 @@ python release.py --help
 
 ### Decide the release type
 
-*Quality-time* adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html), so first you need to decide on the type of release you want to create:
+*Quality-time* adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html), so first you need to decide on the type of release you want to create, conform the [release policy](versioning.md#major-minor-and-patch-releases).
```

Comment:
```suggestion
*Quality-time* adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) so first you need to decide on the type of release you want to create, conform the [release policy](versioning.md#major-minor-and-patch-releases).
```

Reply:
The comma is correct here. See https://quillbot.com/blog/comma-before-so/#:~:text=You%20should%20put%20a%20comma,or%20“for%20this%20reason.”